src/Network/Network/Cortex/VHubRouteTable/NewAzureRmVHubRouteCommand.cs

```diff
@@ -43,6 +43,12 @@ public class NewAzureRmVHubRouteCommand : NetworkBaseCmdlet
         [ValidateNotNullOrEmpty]
         public string NextHop { get; set; }
 
+        [Parameter(
```

Comment:
adding mandatory parameter is breaking change and not allowed in non-breaking change release

Reply:
This is a completely new cmdlet being exposed for the first time, i just missed that parameter in the previous iteration